.eslintrc.json

```diff
@@ -2,7 +2,7 @@
   "root": true,
   "extends": ["./node_modules/sanctuary-style/eslint-es6.json"],
   "parserOptions": {"sourceType": "module"},
-  "env": {"shared-node-browser": true, "es6": true},
```

Comment:
Why can't momi run in the browser?

Reply:
Tricky question; I would have said that `http.createServer` seems like something that only works in the _backend_. But `http` is an external dependency, the user could provide some other sort of library that can create a "server" in the browser, like https://github.com/mafintosh/browser-server. I am slightly confused. Is that what you meant?

https://github.com/fluture-js/momi/blob/6886780c3c12473fb295fc951fecd761c094d1d9/index.js#L145-L153

Reply:
> I would have said that `http.createServer` seems like something that only works in the _backend_.

You are totally right. I just forgot that we actually include http server bindings in this library. They don't necessarily need to be included though - one could use a Middleware (StateFuture) monad to build their Redux middleware, for example. Let's leave it for now.